eu.fortysixandtwo.chessclock.json

```diff
@@ -0,0 +1,67 @@
+{
+    "app-id" : "eu.fortysixandtwo.chessclock",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "master",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "chess-clock",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+	{
+            "name": "gsound",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.gnome.org/GNOME/gsound.git"
```

Comment:
tag or a commit is needed, see https://docs.flatpak.org/en/latest/module-sources.html#git-sources

Reply:
not resolved.

Reply:
sorry,  didn't check closely when resolving.
fixed now.